elasticsearchIndexing/process_changed_mods.py

```diff
@@ -22,30 +22,20 @@
 import os
 import shutil
 import sys
-import typing as t
 
 import requests
 
 from elasticsearchIndexing.build_yindex import build_indices
 from elasticsearchIndexing.es_manager import ESManager
 from utility import log
 from utility.create_config import create_config
-from utility.scriptConfig import Arg, BaseScriptConfig
+from utility.script_config_dict import script_config_dict
+from utility.scriptConfig import BaseScriptConfig
```

Comment:
As we do not use `ScriptConfig` class in individual scripts, then `BaseScriptConfig` can be renamed to `ScriptConfig`